src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -47,6 +54,10 @@
       <Name>ServicesVisualStudio.Next</Name>
       <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
       <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
+      <Ngen>true</Ngen>
```

Comment:
Technically it isn't all. It is just the Assemblies that are:
1. Included in the VSIX
2. Require NGEN

However, we could still define a definition group that only applies to the above.